### ISSUE TITLE
Receiving wrong contentType when idType is set to URI

### ISSUE THREAD

**Issue by JariSavolainen** [wp-graphql/wp-graphql#1850]
I have registered a custom post type. So when I query a page that has same slug/uri as the custom post type, I receive the custom post type instead. I tested this with contentNode as well. I have WP GraphQL Version 1.3.5 installed (WP 5.7).

**So this ...**
```
query getPageByURI {
  page(id: "example", idType: URI) {
    pageId
    title
    contentType {
      node {
        label
      }
    }
  }
  contentNode(id: "example", idType: URI, contentType: PAGE) {
    contentType {
      node {
        label
      }
    }
    ... on Page {
      pageId
    }
    ... on CustomPostType {
      customposttypeId
    }
  }
}
```

**... returns this ...**
```
{
  "data": {
    "page": {
      "pageId": 12345,
      "title": "Example",
      "contentType": {
        "node": {
          "label": "CustomPostTypes"
        }
      }
    },
    "contentNode": {
      "contentType": {
        "node": {
          "label": "CustomPostTypes"
        }
      },
      "customposttypeId": 12345
    }
  }
}
```

**Comment by jasonbahl** [comment]
@JariSavolainen thanks for reporting. This definitely looks like a bug. 

I believe the code path for this is in `/src/Types/RootQuery.php` and then in `/src/Data/NodeResolver.php` if you were interested in trying to chase it down before I can get to it.

**Comment by CptCompiler** [comment]
this does not only happen with custom post types.

I have a post with the uri `test` and no page with this uri.

if I do: 
```
page(id: "/test/", idType: URI)
```

I get the post. Should be null I think

**Comment by jasonbahl** [comment]
@CptCompiler can you share a bit more info?

What happens if you visit `yoursite.com/test` in a browser? What loads? 

Then when you make the GraphQL query for a page by `/test/` uri, what's the response in GraphiQL?

What version of WPGraphQL are you using? The latest release (released yesterday: https://github.com/wp-graphql/wp-graphql/releases/tag/v1.5.2) did include a bugfix for another similar issue (https://github.com/wp-graphql/wp-graphql/pull/1994).

**Comment by CptCompiler** [comment]
Sorry, I updated to the latest version (1.5.2) without checking the old version i had. But it's working now.

**Comment by bhardie** [comment]
Based on the comments above it looks like this was resolved after 1.5.2.  If this is still an issue, please open a new issue referencing this issue.
